specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/applications.json

```diff
@@ -250,6 +257,14 @@
           "type": "integer",
           "format": "int32",
           "description": "The public port to connect to."
+        },
+        "subDomainSuffix": {
+          "type": "string",
+          "description": "The subDomainSuffix of the application and can not greater than 3 characters."
```

Comment:
Please improve the readability of these descriptions (correct grammar, use words instead of writing `subDomainSuffix`). Note you can use the "maxLength" property to enforce the 3 character restriction.

Reply:
> Please improve the readability of these descriptions (correct grammar, use words instead of writing `subDomainSuffix`). Note you can use the "maxLength" property to enforce the 3 character restriction.

Updated. Remove "can not greater than 3 characters" and don't use `maxLength`